### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by cjrd**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/cjrd/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
```
Comment (cjrd) [inline ReadMe.md:1]: Test line note. !fix

Comment (cjrd) [inline ReadMe.md:1]: !fixed

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
```
Comment (cjrd) [inline ReadMe.md:1]: note

**Pull request merged**